### DISCUSSION TITLE
Modeling when an object fails a property

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/linear-algebra#290] [General]
In AT1 (and similar) CheckIt problems, we suggest demonstrating the failure of a linear transformation by picking simple numbers:

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/74f189f2-31fa-4b65-9c6a-416987b0816d)

In the sample solution (thanks @siwelwerd!) Drew did more algebra:

![image](https://github.com/TeamBasedInquiryLearning/linear-algebra/assets/1559632/b50e097f-30dc-4715-9829-d9665431b43e)

What's the community's preference here? My hot take: not only is the algebra making things more difficult than necessary, I don't like statements like `expression \neq other_expression` because expressions can look different while still being equal (but simplified real numbers usually don't).

**Comment by siwelwerd** [comment]
Benefits of the approach in the sample solution: 

1. You don't have to decide a priori whether you think it is or is not a linear transformation--just start checking and see what shakes out.
2. This mirrors the approach in showing that something *is* a linear transformation.
3. Everything in these problems are polynomials, and it is straightforward to verify when two polynomials are the same.

**Comment by fragandi** [comment]
Personally, I would combine the two in the sample solution and add a numerical counterexample at the end. This would accomplish
1. Use the same algebraic approach for both tasks
2. Emphasise "number sense" in making sure what they say makes sense with some choice of numbers

I would also remark at the end of the solution that a numerical counterexample **is enough** to prove that the property fails, but you might need the algebra or lots of guess and check to find such counterexample!

**Reply by StevenClontz** [reply]
Actually, in general I think I like the idea of supplying multiple valid solutions to readers, to emphasize that there are many ways of writing a valid solution (not just "what do you want me to say here").

**Comment by sharonakrinsky** [comment]
My students are not really getting the point when they try to do it algebraically, so I make them use the explicit example for the contradiction. It really drives home the do it one way get one answer, do it the other way get a different answer in a concrete way. So I would definitely like to see at least one numerical example in the sample. But I also second including both.
